adapters/clients/db_users.go

```diff
@@ -0,0 +1,78 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package clients
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	enterrors "github.com/weaviate/weaviate/entities/errors"
+	"github.com/weaviate/weaviate/usecases/auth/authentication/apikey"
+)
+
+type RemoteUser struct {
+	client       *http.Client
+	nodeResolver nodeResolver
+}
+
+type nodeResolver interface {
+	NodeHostname(nodeName string) (string, bool)
+}
+
+func NewRemoteUser(httpClient *http.Client, nodeResolver nodeResolver) *RemoteUser {
+	return &RemoteUser{client: httpClient, nodeResolver: nodeResolver}
+}
+
+func (c *RemoteUser) GetUserStatus(ctx context.Context, nodeName string, users map[string]time.Time, returnStatus bool) (*apikey.UserStatusResponse, error) {
+	p := "/cluster/users/db"
```

Comment:
```suggestion
	p := "/cluster/users/db/"
```
nit: for consistency with other occurrences of the path (makes it more searchable)
- https://github.com/weaviate/weaviate/pull/7786/files#diff-133c32f1954ae34d9b7a6e8957509231404411b75c308a6a47b43620d291489bR40
- https://github.com/weaviate/weaviate/pull/7786/files#diff-111130665d0db0ff03380fd5f47b01932a4190297e358b5e7f0a84fc555a200fR44

Reply:
see here: https://github.com/weaviate/weaviate/pull/7786#discussion_r2041721067